Dockerfile

```
@@ -4,30 +4,25 @@ LABEL org.opencontainers.image.source=https://github.com/svtechnmaa/BNGBlaster_w
 
 LABEL maintainer "linhnt-hub"
 
+WORKDIR /BNGBlaster_web_client
+ENV BNGBLASTER_CONFIG="/root/default_variable.yml"
+
 RUN apt-get update -y \
```

Hadolint (2.14.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)